Source/Schema.NET/Schema.NET.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup Label="Package References">
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Text.Json" Version="6.0.0-preview.7.21377.19" />
```

Comment:
We have to wait for .NET 6 to be released before merging right?

Reply:
I mean, this is actually compiling with the .NET 5 SDK. I'd be fine having it merged prior to the release of .NET 6 but it does limit any other releases we might want to do between now and then.

If that isn't much of an issue, I think perhaps we merge and release a preview version of Schema.NET with S.T.J to see what issues etc crop up that we've missed?

Reply:
I think it's worth it to update github actions to the .NET 6 SDK.

Note: you can every easily do it with my version of setup-dotnet that is far simpler than setup-dotnet called setup-latest-dotnet 😂. Yes I made it for the reason of getting the daily quality builds for github actions based on a specific major and minor version feed name for .NET.

Reply:
@AraHaan We only need to update the `global.json` file. I've setup everything (GitHub actions, Azure Pipelines, Appveyor) to get the SDK version from there.